src/platform/forms-system/src/js/components/ProgressButton.jsx

```diff
@@ -40,6 +43,9 @@ class ProgressButton extends React.Component {
         id={`${this.id}-continueButton`}
         onClick={this.props.onButtonClick}
         aria-label={this.props.ariaLabel || null}
+        aria-describedby={
+          !environment.isProduction() && this.props.ariaDescribedBy
```

Comment:
In the long run, I think adding the production check to the `config/form.js` would be easier than adding it here. Once the change has been evaluated, you only need to remove it in your owned code and won't need to wait for another team to review the change.

Reply:
@Mottie As a noob to this codebase, digging into the tests were a bit challenging! That being said I think I have this in a decent place. I tested with and without a form.js tag (simulating what would be elsewhere in the app) and the `aria-described` tags failover properly to not being rendered if there's nothing in form.js attribute wise.